common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -214,6 +217,9 @@ void extractState(@NonNull final Bundle state) {
         mAuthIntent = state.getParcelable(AUTH_INTENT);
         mPkeyAuthStatus = state.getBoolean(PKEYAUTH_STATUS, false);
         mAuthorizationRequestUrl = state.getString(REQUEST_URL);
+        if (mAuthorizationRequestUrl != null) {
+            isBrokerRequest = checkBrokerRequest(mAuthorizationRequestUrl);
```

Comment:
Can we instead check if we are inside :auth process? I think we do that in some parts of the code

Reply:
I'll look into this, that would be better